packages/core/src/lib/implementation/json-to-gql.ts

```diff
@@ -71,13 +71,15 @@ export function jsonToGql(report: Report) {
   >;
 }
 
-function transformSeverity(severity: Issue['severity']): IssueSeverity {
+function transformSeverity(severity: CliIssueSeverity): IssueSeverity {
   switch (severity) {
     case 'info':
       return IssueSeverity.Info;
     case 'error':
       return IssueSeverity.Error;
     case 'warning':
       return IssueSeverity.Warning;
+    default:
+      throw new Error(`Severity ${severity} unknown`);
```

Comment:
I would omit the default case. The reason is that when we only have known cases and we extend the possible values, then TypeScript will tell us we're missing a case (thanks to `noImplicitReturns` flag) and we'll be forced to fix it at build time (instead of shipping a runtime error).

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/e7b1ac9a-169d-4fe5-9725-7757c6e989aa)